File: 8_kyu/Python/positive_sum.py
def positive_sum(arr):
    total = 0
    for i in arr:
        if i > 0:
            total += i
    return total






print(positive_sum([1,2,3,4,5]))  # 15
print(positive_sum([1,-2,3,4,5]))  # 13
print(positive_sum([-1,2,3,4,-5]))  # 9

File: 8_kyu/JavaScript/positiveSum.js
function positiveSum(arr) {
  sum = 0;
  for (let i = 0; i < arr.length +1; i++) {
    if (arr[i] > 0) {
      sum += arr[i];
    }
  }
  return sum;
}



console.log(positiveSum([1,2,3,4,5])) // 15
console.log(positiveSum([1,-2,3,4,5])) // 13
console.log(positiveSum([])) // 0
console.log(positiveSum([-1,-2,-3,-4,-5])) // 0
console.log(positiveSum([-1,2,3,4,-5])) // 9